test/test_reconfig/test_reconfig.cpp

```diff
@@ -0,0 +1,208 @@
+/**

```

Comment:
maybe we should leave this file where it was originally, now that you have it in a TEST